sparkly/writer.py

```diff
@@ -56,6 +74,7 @@ def by_url(self, url):
             - Elastic ``elastic://``
             - MySQL ``mysql://``
             - Parquet ``parquet://``
+            - Redis ``redis://`` or ``rediss://``
```

Comment:
`redis:/` and `rediss://` URLs are parsed by redis-py to set up the connection type etc. `rediss://` creates an SSL wrapped TCP socket connection. Details: http://www.iana.org/assignments/uri-schemes/prov/rediss